frontend/src/components/survey-handler/survey-handler.tsx

```diff
@@ -25,7 +25,10 @@ interface SurveyPageProps {
   survey: Survey;
 }
 
-const questionTypes = {
+const questionTypes: Record<
+  QuestionTypeName,
+  typeof SingleChoice | typeof MultipleChoice
```

Comment:
I think this doesn't scale very well as we're going to have more question types in the future. Maybe it's interesting to transform into a type or simply use a more generic type like ReactNode or JSX.Element or even create a generic interface that has the base props of a question.